AdminWebsite/AdminWebsite/Mappers/HearingTypeResponseMapper.cs

```diff
@@ -0,0 +1,16 @@
+using AdminWebsite.Contracts.Responses;
+using BookingsApi.Contract.V2.Responses;
+
+namespace AdminWebsite.Mappers;
+
+public static class HearingTypeResponseMapper
+{
+    public static HearingTypeResponse Map(this CaseTypeResponseV2 response) =>
+        new()
+        {
+            Group = response.Name,
```

Comment:
Do we still need to use group? Feels like hang over from V1. How much would it be to rename group to name so the prop names line up and are up to date in terms of what they represent?

Reply:
I've created a separate ticket for this. There's still a bit of terminology hangover: https://tools.hmcts.net/jira/browse/VIH-11250